codecov_cli/commands/labelanalysis.py

```diff
@@ -55,13 +56,23 @@
 @click.option(
     "--dry-run",
     "dry_run",
```

Comment:
It's not clear (to me) what `label-analysis` does on a "wet" run. Add a function docstring or else pass `@command(help=...)` to help users understand.

Reply:
I suppose you are right... possibly we should have not built it this way, but created a new runner to do what the `dry-run` option does. This would reinforce the intended way of using it